website/docs/advanced/targeting.md

```diff
@@ -194,6 +194,67 @@ The exact value that will be served to the users that fall into that fraction. D
 | Whole Number   | Integer      | any whole number within the range of `Int32`      |
 | Decimal Number | Double       | any decimal number within the range of `double` |
 
+## Stickiness & Consistency
+
+The percentage-based targeting is sticky by design and consistent across all SDKs.
+
+Percentage-based targeting is based on the identifier passed in the `User Object` to get `getValue()` methods in the SDKs. 
+The SDKs are hashing the identifier concatenated with the evaluated Feature Flag's key and assign a 0-99 number to the User for a specific Feature Flag. 
```

Comment:
The SDKs are hashing the concatenated value of the User Object's identifier and the evaluated Feature Flag's key. The SDK uses this hash to choose a number on a 0-99 scale and binds it to the given User. Then, this number is used to evaluate a particular Feature Flag's value based on it's configured rules.

Reply:
Fixed